bedrock/firefox/views.py

```diff
@@ -186,29 +186,29 @@ def sign_attribution_codes(codes):
 @csrf_exempt
 def send_to_device_ajax(request):
     locale = l10n_utils.get_locale(request)
-    phone_or_email = request.POST.get('phone-or-email')
+    email = request.POST.get('s2d-email')
 
     # ensure a value was entered in phone or email field
-    if not phone_or_email:
-        return JsonResponse({'success': False, 'errors': ['phone-or-email']})
+    if not email:
+        return JsonResponse({'success': False, 'errors': ['s2d-email']})
 
     # pull message set from POST (not part of form, so wont be in cleaned_data)
     message_set = request.POST.get('message-set', 'default')
 
     # begin collecting data to pass to form constructor
-    data = {'platform': request.POST.get('platform')}
-
-    # determine if email or phone number was submitted
-    data_type = 'email' if '@' in phone_or_email else 'number'
+    data = {
+        'platform': request.POST.get('platform'),
+        'email': email,
+    }
 
     # populate data type in form data dict
-    data[data_type] = phone_or_email
+    data[data_type] = email
 
     # instantiate the form with processed POST data
     form = SendToDeviceWidgetForm(data)
```

Comment:
This form can also be cleaned up to remove the `number` field and the `clean_number` method.

https://github.com/mozilla/bedrock/blob/bbd6f42a9bf022097d83a1758cadec7f0c2fbf0a/bedrock/firefox/forms.py#L11